### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vincent-pli**
This is a minimal steps which could make Operator works, draft version, please add item missing, thanks

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,53 @@
...
-> software. Previously this knowledge only resided in the minds of
-> administrators, various combinations of shell scripts or automation
-> software like Ansible. It was outside of your Kubernetes cluster and
-> hard to integrate. With Operators, we changed that.
+Kubernetes [Operator](https://operatorhub.io/getting-started) to manage installations, updates and removal of Tektoncd projects (pipeline, dashboard, …)
+
+The following steps will install [Tekton Pipeline](https://github.com/tektoncd/pipeline) and configure it appropriately for your cluster.
+1. Create namespace: `tekton-operator`  
+
+    `kubectl create namespace openshift-pipelines-operator`
```
Comment (nikhil-thomas) [inline README.md]: `kubectl create namespace tekton-operator`

File: README.md [hunk 2]
```diff
@@ -1,11 +1,53 @@
...
+    
+2. Apply Operator crd  
+
+    `kubectl apply -f deploy/crds/*_crd.yaml`
+    
+3. Deploy the Operator
```
Comment (nikhil-thomas) [inline README.md]: before deploying the operator image should be built. The image should be specified in the `deploy/operator.yaml` deployment manifest

Comment (vincent-pli) [reply README.md]: Yes, this is a problem. I think we need build an official image for it.

File: README.md (hunk 1 above)
Comment (vincent-pli) [reply README.md]: Thanks, my mistake

File: README.md (hunk 2 above)
Comment (vincent-pli) [reply README.md]: I log a issue #16 to trace it

File: README.md [hunk 3]
```diff
@@ -1,11 +1,53 @@
...
+    `operator-sdk build ${YOUR_REGISTORY}/openshift-pipelines-operator:${IMAGE_TAG}`
+    
+3. Push the container image  
+
+    `docker push ${YOUR-REGISTORY}/openshift-pipelines-operator:${IMAGE-TAG}`
+    
+4. Edit the 'image' value in deploy/operator.yaml to match to your image  
+
+## The CRD
+This is a sample of [crd](https://github.com/tektoncd/operator/blob/master/deploy/crds/operator_v1alpha1_config_cr.yaml)
```
Comment (houshengbo) [inline README.md]: I remember we have a demo saying we can either go with the auto-install or manual install.
Shall we add the instruction on it?

File: README.md [hunk 4]
```diff
@@ -1,11 +1,53 @@
...
+    `kubectl create namespace tekton-operator`
+    
+2. Apply Operator crd  
+
+    `kubectl apply -f deploy/crds/*_crd.yaml`
+    
+3. Deploy the Operator  
+
+    `kubectl -n tekton-operator apply -f deploy/`  
+
```
Comment (houshengbo) [inline README.md]: I remember we have a demo saying we can either go with the auto-install or manual install of CR?
Shall we add the instruction on it?

Comment (vincent-pli) [reply README.md]: Done

**Pull request merged**